pubsub2/fuzz_record_list/fuzz.go

```diff
@@ -0,0 +1,53 @@
+package fuzz
```

Comment:
Just to confirm, the fuzz package is for testing, correct?

Reply:
Yep